cmd/kpod/info.go

```diff
@@ -0,0 +1,200 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"runtime"
+
+	"github.com/docker/docker/pkg/system"
+	"github.com/ghodss/yaml"
+	"github.com/urfave/cli"
+)
+
+var (
+	infoDescription = "display system information"
+	infoCommand     = cli.Command{
+		Name:        "info",
+		Usage:       infoDescription,
+		Description: infoDescription,
```

Comment:
Any change you could give a better description then just repeating the Usage message.  I like multi-line descriptions usually stolen from the man page extended description.  You could probably steal some of you text from your commit message.